package.json

```diff
@@ -6,15 +6,10 @@
   "scripts": {
```

Comment:
All URLs and strings should be added to constants.js

Reply:
Done!